torch/nn/parallel/distributed.py

```diff
@@ -187,7 +212,14 @@ def _sync_params(self):
         if len(buffers) > 0:
             # cross-node buffer sync
             flat_buffers = _flatten_dense_tensors(buffers)
-            dist.broadcast(flat_buffers, 0)
+
+            if dist._backend == "nccl":
+                # TODO: deprecate this broadcast along with
+                # dist.all_reduce_multigpu
+                dist.broadcast(flat_buffers, 0, self.bcast_grp)
```

Comment:
do all NCCL calls need an explicitly specified group? why can't you just use the default one like in all other backends?

Reply:
I'm sorry I didn't get that